css/styles.css

```diff
@@ -0,0 +1,215 @@
+    95% 25%,
+    80% 8%,
+    15% 10%;
+}
+
+.flex-navbar {
+  background-color: #fff;
+  color: white;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  font-size: 20px;
+}
+
+.flex-nav-list {
+  list-style-type: none;
+  margin-right: 30%;
+}
+
+.flex-nav-list .list-item {
+  display: inline-block;
+  padding: 20px 10px;
+}
+
+.flex-navbar a {
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+}
+
+.flex-navbar a:hover {
+  color: #16d026;
+}
+
+.primary-logo {
+  height: 60px;
+}
+
+.menu-line {
+  width: 30px;
+  height: 3.5px;
+  background-color: #000;
+  margin-bottom: 5px;
+}
+
+#headline-section {
+  height: 916px;
+  margin-left: 18%;
+  margin-right: 18%;
+  margin-top: 150px;
+  background-image: none;
+}
+
+.headline-title {
+  font-weight: bold;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  text-align: center;
+  margin: 0 24px;
+}
+
+#headline-section p {
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #172b4d;
+  margin: 12px 24px 0 24px;
+  text-align: center;
+}
+
+#contact ul {
+  display: flex;
+  align-items: center;
+  padding-left: 0;
+  margin-left: 24px;
+}
+
+.icon {
+  list-style-type: none;
+  margin-right: 20px;
+  margin-left: 40%;
+}
+
+h3 {
+  margin-top: 12px;
+  margin-bottom: 0;
+  font-weight: bold;
+  font-size: 32px;
+  color: #172b4d;
+  line-height: 44px;
+}
+
+footer h3 {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #000;
+  margin-left: 24px;
+}
+
+footer h3:hover {
+  text-decoration: none;
+  text-decoration-thickness: 2px;
+  text-decoration-color: #159c06;
+}
+
+#headline-section footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.menu {
+  display: none;
+}
+
+@media screen and (max-width: 768px) {
+  .flex-nav-list {
+    display: none;
+  }
+
+  .menu {
+    display: block;
+    position: absolute;
+    right: 25px;
+    top: 25px;
+  }
+
+  .index-page {
+    background-image:
+      url("../img/piramid.png"),
+      url("../img/square.png"),
+      url("../img/revertsquare.png"),
+      url("../img/elipsir.png"),
+      url("../img/artrock.png");
+    background-repeat: no-repeat;
+    background-position:
+      -15% 35%,
+      -15% 84%,
+      135% 85%,
+      122% 25%,
+      -15% 10%;
+  }
+
+  .primary-logo {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 505px) {
+  .index-page {
+    background: white;
+  }
+}
+
+@media screen and (max-width: 375px) {
+  .menu {
+    display: none;
+  }
+
+  .flex-nav-list {
+    display: block;
+    list-style-type: none;
+    width: 100%;
+    text-align: center;
+    padding-top: 10px;
+    margin-left: -20px;
+  }
+
+  .flex-nav-list .list-item {
+    display: block;
+    margin-right: -100px;
+    border-top: 1px solid rgb(0, 0, 0);
+  }
+
+  .primary-logo {
+    width: fit-content;
+    margin-left: 10px;
+  }
+
+  #headline-section {
+    margin-top: 30px;
+  }
+}
```

Comment:
- [ ] Although you have done an excellent job designing the mobile version of your project :+1:, it is important to note that as a project requirement, it should be mobile-first. This means that you should avoid using media queries to adapt the design to different screen sizes after the fact, and instead focus on designing for mobile screens as the primary consideration. The reason for this is that it can lead to a more streamlined and efficient design process, as well as improved user experience on mobile devices :+1:.
- [ ] In addition, it is mandatory that the minimum screen size for mobile devices should be 375px, so please ensure that there are no design flaws at that size, as illustrated in the video below (nav bar errors).


https://user-images.githubusercontent.com/105117832/227159380-f1169b02-3ab6-461c-ab98-2d97baa6c8d6.mp4

**Required design**

![Screenshot from 2023-03-23 04-26-19](https://user-images.githubusercontent.com/105117832/227159794-fda4bd49-7621-42c1-8d58-3302d951e862.png)